src/Command/App/NewCommand.php

```diff
@@ -76,13 +84,31 @@ protected function commandRequiresAuthentication(InputInterface $input): bool {
     return FALSE;
   }
 
+  /**
+   * @param string $dir
+   *
+   * @throws \Exception
+   */
+  protected function createNextJsProject(string $dir): void {
```

Comment:
Make this private unless you know it's going to be used somewhere else

Reply:
This is updated!